### PR TITLE
chore: enable dependabot

### PR DESCRIPTION
**Pull request by josokinas**
## Purpose

Dependabot is used to keep dependencies up to date.

## Approach

Configuring Dependabot based on: https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates

## Testing

We will have to wait and see if PRs are being opened.

## Risks

N/A

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+version: 2
+
+updates:
+  - package-ecosystem: 'github-actions'
+    directory: '/'
+    schedule:
+      interval: 'weekly'
```
Comment (rabelloo) [inline .github/dependabot.yml:7]: Interesting, why weekly and not daily? 🤔

**Review by rabelloo — APPROVED** [review]
LGTM.

I've always found that the default dependabot settings are quite sensible, and it always worked well for me.

Just curious, did you find something you disliked?

File: .github/dependabot.yml (hunk 1 above)
Comment (josokinas) [reply .github/dependabot.yml:7]: I feel like we don't care much about having actions updated, also if they would require workflow changes - less likely we will need to address ASAP. If to dedicate some time once a week (if any), IMO is enough?

P.S. We can also see how often such occur and change the preference, if we see we'd need to.

**Comment by josokinas** [comment]
> Just curious, did you find something you disliked?

What do you mean, when shaping up the script? It was pre-filled TBH, I only added the GitHub Actions one.

Well, comparing to what we had previously, have a setting on repository itself is so much better!

**Comment by rabelloo** [comment]
> What do you mean, when shaping up the script? It was pre-filled TBH, I only added the GitHub Actions one.

I mean Dependabot works well even without that file `.github/dependabot.yml`, i.e. it's zero-config.

**Comment by josokinas** [comment]
> I mean Dependabot works well even without that file `.github/dependabot.yml`, i.e. it's zero-config.

Really? But it shows this: https://github.com/onfido/castor-icons/network/updates

**Comment by rabelloo** [comment]
> Really? But it shows this: https://github.com/onfido/castor-icons/network/updates

Ah, it needs the config for Insights then, makes total sense.
It does create PRs without it though, I'm sure.

**Comment by josokinas** [comment]
> It does create PRs without it though, I'm sure.

I looked this up a little, seems like previously it was [an app you had to install and setup](https://github.com/marketplace/dependabot-preview), but now after Dependabot was integrated directly into GitHub I believe having a config is [recommended way of enabling it](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/enabling-and-disabling-version-updates).

**Pull request merged**